simple/sample/output/process/report.json

```diff
@@ -1,37 +1,37 @@
 {
   "status": "SUCCESS",
-  "startTime": "2023-10-23 10:50:22.644858",
-  "lastUpdate": "2023-10-23 10:50:30.534262",
+  "startTime": "2023-10-23 14:43:10.816166",
```

Comment:
shall we gitignore this file?

Reply:
Let's keep it around for now until we have reached a steady state.

One way we can address it is to have a mode that persists it with fixed timestamps for samples and tests.